src/nvim/testdir/test_signs.vim

```diff
@@ -133,9 +133,8 @@ func Test_sign()
   call assert_fails("sign define Sign4 text=a\e linehl=Comment", 'E239:')
   call assert_fails("sign define Sign4 text=\ea linehl=Comment", 'E239:')
 
-  " Only 1 or 2 character text is allowed
+  " Only 0, 1 or 2 character text is allowed
   call assert_fails("sign define Sign4 text=abc linehl=Comment", 'E239:')
-  call assert_fails("sign define Sign4 text= linehl=Comment", 'E239:')
```

Comment:
It's fine to update comments to reflect new behavior but line deletions should be avoided in legacy tests.

Reply:
But this will not fail anymore.
I could add a test instead to assert that it doesn't fail, but I already did this in the lua test.

Reply:
Comment it out.

Reply:
Ah I see what you mean. Updated